ckanext/iati_generator/plugin.py

```diff
@@ -2,14 +2,51 @@
 import logging
 from ckan import plugins as p
 from ckan.plugins import toolkit
+from flask import Blueprint, Response
+
+# Librería IATI
+from okfn_iati import (
+    Activity, Narrative, OrganizationRef, IatiXmlGenerator,
+    ActivityStatus, OrganisationType, IatiActivities
+)
 
 log = logging.getLogger(__name__)
 
 
 class IatiGeneratorPlugin(p.SingletonPlugin):
     p.implements(p.IConfigurer)
+    p.implements(p.IBlueprint)
 
     def update_config(self, config_):
         toolkit.add_template_directory(config_, "templates")
         toolkit.add_public_directory(config_, "public")
         toolkit.add_resource("assets", "ckanext-iati-generator")
+
+    def get_blueprint(self):
+        iati_bp = Blueprint("iati_generator", __name__)
+
+        @iati_bp.route("/iati/generate")
```

Comment:
Blueprints goes at a new `blueprint` folder
This PR do not require any blueprint (so far)
Just install the library in requirements